src/screens/Analytics/Logs/DisputeLogs/DisputeLogs.res

```
@@ -1,151 +1,19 @@
-  | SDK => "Event"
-  | WEBHOOKS => ""
-  }
-
-  let codeBlock =
-    <UIUtils.RenderIf
-      condition={logDetails.response->isNonEmptyString || logDetails.request->isNonEmptyString}>
-      <div
-        className="flex flex-col gap-4 border-l-1 border-border-light-grey show-scrollbar scroll-smooth overflow-scroll px-5 py-3 w-3/5">
-        <UIUtils.RenderIf
-          condition={logDetails.request->isNonEmptyString &&
-            selectedOption.optionType !== WEBHOOKS}>
-          <PrettyPrintJson
-            jsonToDisplay=logDetails.request
-            headerText={requestHeader->Some}
-            maxHeightClass={logDetails.response->String.length > 0 ? "max-h-25-rem" : ""}
-          />
-        </UIUtils.RenderIf>
-        <UIUtils.RenderIf condition={logDetails.response->isNonEmptyString}>
-          <PrettyPrintJson jsonToDisplay={logDetails.response} headerText />
-        </UIUtils.RenderIf>
-      </div>
-    </UIUtils.RenderIf>
-
-  open OrderUtils
-  <PageLoaderWrapper
-    screenState customUI={<NoDataFound message="No logs available for this dispute" />}>
-    {if disputeId->HSwitchOrderUtils.isTestData || data->Array.length === 0 {
-      <div
-        className="flex items-center gap-2 bg-white w-full border-2 p-3 !opacity-100 rounded-lg text-md font-medium">
-        <Icon name="info-circle-unfilled" size=16 />
-        <div className={`text-lg font-medium opacity-50`}>
-          {"No logs available for this dispute"->React.string}
-        </div>
-      </div>
-    } else {
-      <Section
-        customCssClass={`bg-white dark:bg-jp-gray-lightgray_background rounded-md pt-2 pb-4 flex gap-7 justify-between h-48-rem !max-h-50-rem !min-w-[55rem] max-w-[72rem] overflow-scroll`}>
-        {timeLine}
-        {codeBlock}
-      </Section>
-    }}
-  </PageLoaderWrapper>
+  <AuditLogUI
+    id={paymentId}
+    promiseArr={[
+      fetchDetails(disputesLogsUrl),
+      fetchDetails(webhooksLogsUrl),
+      fetchDetails(connectorLogsUrl),
+    ]}
```

Can't we move all this API Call to AuditLogUI as nothing happening in this component except passing the prop

---

functions can also can be post or get so its been passed from parent componet itself